proto/codec_config.proto

```diff
@@ -16,13 +16,18 @@ message DecoderConfigLpcm {
   optional uint32 sample_rate = 3;
 }
 
+message OpusEncoderMetadata {
+  optional int32 target_bitrate = 1;
+}
+
 message DecoderConfigOpus {
   optional uint32 version = 1;
   optional uint32 output_channel_count = 2;
   optional uint32 pre_skip = 3;
   optional uint32 input_sample_rate = 4;
   optional int32 output_gain = 5;
   optional uint32 mapping_family = 6;
+  optional OpusEncoderMetadata opus_encoder_metadata = 7;
```

Comment:
Why is this in DecoderConfigOpus?

Reply:
It is metadata used to encode the streams themselves. It does not directly show up in the IAMF bistream, only indirectly because it controls the encoder's output.

For example libopus allows selecting the target bitrate and application (VOIP, AUDIO, etc.). Controlling these knobs would result in different audio frames based on whatever Opus outputs.

I put it with the Codec Config because for future compatibility I imagined perhaps a user would want to encode an Opus Audio stream with an Opus VOIP with different `Codec Config` settings.  Of course this is only relevant in enhanced profile.

Maybe the artist wants to play a song and comment about it live and encode it all in one IAMF stream with 2 separate Audio Elements, 2 Codec Configs and slightly different metadata used to encode the Opus streams with libopus.